flow/layers/color_filter_layer.cc

```diff
@@ -45,23 +48,29 @@ void ColorFilterLayer::Paint(PaintContext& context) const {
   TRACE_EVENT0("flutter", "ColorFilterLayer::Paint");
   FML_DCHECK(needs_painting(context));
 
-  AutoCachePaint cache_paint(context);
-
   if (context.raster_cache) {
+    AutoCachePaint cache_paint(context);
     if (layer_raster_cache_item_->IsCacheChildren()) {
-      cache_paint.setColorFilter(filter_);
+      cache_paint.setColorFilter(filter_->skia_object());
```

Comment:
Can we switch these all to use the non-skia version of cache_paint.setColorFilter()?

Reply:
Using DlColorFilter